### PR TITLE
MAR-4610: Upgrade to Python 3

### PR DESCRIPTION
**Pull request by Reshma-M-A**

### CODE REVIEW

**Review by Revit — APPROVED** [review]

File: dmapiclient/base.py [hunk 1]
```diff
@@ -1,10 +1,7 @@
-from __future__ import absolute_import
 import logging
-
-try:
-    import urlparse
-except ImportError:
-    import urllib.parse as urlparse
+import urllib.request
```
Comment (Reshma-M-A) [inline dmapiclient/base.py:2]:
```suggestion
import urllib.request
```
Should we remove this? We don't seem to use `urllib.request`

File: dmapiclient/base.py [hunk 2]
```diff
@@ -1,10 +1,7 @@
...
+import urllib.parse
+import urllib.error
```
Comment (Reshma-M-A) [inline dmapiclient/base.py:4]:
```suggestion
import urllib.error
```
Should we remove this? We don't see to use `urllib.error`

File: dmapiclient/base.py (hunk 1 above)
Comment (tobiasbrown) [reply dmapiclient/base.py]: Yes, it looks like this import was added by the [fixer](https://github.com/AusDTO/dto-digitalmarketplace-apiclient/commit/b91ebae37a11de070911f2695be52b9897397ade#diff-846a0177861852a2fdc309db3dc53ce16dd85a70d73758fe22adcbcc83972ed2).

File: dmapiclient/base.py (hunk 2 above)
Comment (tobiasbrown) [reply dmapiclient/base.py]: Yes, it looks like this import was added by the [fixer](https://github.com/AusDTO/dto-digitalmarketplace-apiclient/commit/b91ebae37a11de070911f2695be52b9897397ade#diff-846a0177861852a2fdc309db3dc53ce16dd85a70d73758fe22adcbcc83972ed2).

**Review by tobiasbrown — APPROVED** [review]

**Review by Reshma-M-A — COMMENTED** [review]
Changes looks good!

**Pull request merged**